src/System.Net.Security/src/project.json

```diff
@@ -3,7 +3,8 @@
     "System.Diagnostics.Contracts": "4.0.0-*",
     "System.Threading": "4.0.0-*",
     "System.Net.Primitives": "4.0.10-beta-*",
-    "System.Security.Cryptography.OpenSsl": "4.0.0-beta-*"
+    "System.Security.Cryptography.OpenSsl": "4.0.0-beta-*",
```

Comment:
Well, this is in the implementation details, not the contract, so there's no exposure from it.

But, that said, I should have split the project.json into Windows vs Unix, like I did for X509.  I'll do that as a followup.